pkg/test/test.go

```diff
@@ -38,22 +45,23 @@ func Boot(configFilenames ...string) {
 		bootFromFile(filename)
 	}
 
-	wait.Wait()
```

Comment:
i think we can re-add this and remove the implementation at the bottom where we wait for each mock individually. This should then increase the overall speed

Reply:
✅